app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -276,7 +276,7 @@ class TopicLessonsFragmentTest {
   }
 
   @Test
-  fun testLessonsPlayFragment_loadRatiosTopic_clickExpandListIconIndex1_clickExpandListIconIndex2_chapterListForIndex1IsNotDisplayed() { // ktlint-disable max-line-length
+  fun testLessonsPlayFragment_clickExpandListIconIndex1then2_chapterListForIndex1NotDisplayed() {
```

Comment:
Its nice but the `T` should be capital in `1then2`.

`testLessonsPlayFragment_clickExpandListIconIndex1Then2_chapterListForIndex1NotDisplayed `

Reply:
Ditto at other places.

Reply:
@varunvjha Always reply to these comments.